netket/experimental/dynamics/_rk/_solver.py

```diff
@@ -11,35 +11,29 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
+from .._solver import (
+    AbstractSolver,
+    SolverState,
+    append_docstring,
+    args_adaptive_docstring,
+    args_fixed_dt_docstring,
+)
 
-from collections.abc import Callable
+from typing import Callable
 
 import jax
 import jax.numpy as jnp
 
-from netket.utils.struct import dataclass
-from netket.utils.types import Array, PyTree
+from netket.utils.types import Array
+from .._utils import expand_dim
+from netket.utils.struct import field
+from ._tableau import TableauRKExplicit
```

Comment:
The ordering for imports should loosely be
standard libraries

other packages

netket global imports

local imports

```suggestion
from netket.utils.types import Array
from netket.utils.struct import field

from .._utils import expand_dim
fom .._solver import ( ...)
from ._tableau import TableauRKExplicit
```